core/src/trezor/lvglui/scrs/template.py

```
@@ -177,7 +177,7 @@
 
 
 class BTCDeriveSelectionScreen(FullSizeWindow):
-    def __init__(self, prev_scr=None, addr_type=None, net_scr=None):
+    def __init__(self, prev_scr=None, addr_type=None, net_scr=None, has_taproot=True):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)